File: 1791-RichestCustomerWealth/1791-RichestCustomerWealth.py
# Last updated: 9/16/2025, 12:52:25 AM
class Solution(object):
    def maximumWealth(self, accounts):
        """
        :type accounts: List[List[int]]
        :rtype: int
        """
        ma = 0 
        for i in accounts:
            sum = 0
            for j in i:
                sum = sum + j;
            ma = max(ma,sum)

        return ma
        

File: 1791-RichestCustomerWealth/1791-RichestCustomerWealth.cpp
// Last updated: 2/13/2026, 12:56:24 PM
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) 
    {
        int cnt = 0;

        for(int i = 0; i<accounts.size(); i++)
        {
            int c = accumulate(accounts[i].begin(), accounts[i].end(), 0);
            cnt = max (c,cnt);
        }

        return cnt;
    }
};